schemas/Identifier.json

```diff
@@ -1,7 +1,7 @@
 {
   "$schema": "http://json-schema.org/draft-04/schema#",
   "title": "Identifier",
-  "description": "An identifier for a user or institution.",
+  "description": "An identifier for a user or institution. Between 2-20 characters inclusively. May include dashes (-), but not at the beginning or end and no more than one consecutively.",
```

Comment:
I think UUIDs should be valid as well, which would mean the length would need to be longer